fuzzbucket_client/__main__.py

```diff
@@ -501,6 +518,7 @@ def main(sysargs: typing.List[str] = sys.argv[:]) -> int:
     parser_delete_key.set_defaults(func=client.delete_key)
 
     known_args, unknown_args = parser.parse_known_args(sysargs[1:])
+    known_args = _normalize_known_args(known_args)
```

Comment:
You might consider using a custom `type` for the `user` argument, as described [here](https://docs.python.org/3/library/argparse.html#type). That would avoid the need to clone the namespace to update the username.

Reply:
I had a version of this where I'd used `type` 😂  but having the logging side effect in there felt weird. WDYT?

Reply:
I'm OK either way